tests/lib/snaps/store/test-snapd-refresh-control-provider.v1/meta/snap.yaml

```diff
@@ -0,0 +1,17 @@
+name: test-snapd-refresh-control-provider
```

Comment:
if you create a snap in tests/lib/snaps/store/ then it means that snap will uploaded to the store, in this case we need to create the snapcraft.yaml for the snaps ready for any architecture.

Reply:
Done.